src/feature/Users/index.jsx

```diff
@@ -1,9 +1,102 @@
 import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faSearch, faPlus } from "@fortawesome/free-solid-svg-icons";
+
+import EmployeeTable from "../../components/EmployeeTable";
+import styles from "./Users.module.css";
+
+const employeesData = [
+  {
+    id: 1,
+    name: "Joshua Gideon",
+    email: "joshgideon.com",
+    role: "Super Admin",
+    creationDate: "2017-01-02",
+    position: "CEO and Founder",
+    avatar: "images/img5.jpg",
+  },
+  {
+    id: 2,
+    name: "Henry Solomon",
+    email: "henrysolomon@gmail.com",
+    role: "Admin",
+    creationDate: "2020-11-04",
+    position: "Web developer",
+    avatar: "images/img3.png",
+  },
+  {
+    id: 3,
+    name: "Michael Orji",
+    email: "michaelorji.com",
+    role: "Admin",
+    creationDate: "2021-04-02",
+    position: "Graphics designer",
+    avatar: "images/img1.jpg",
+  },
+  {
+    id: 4,
+    name: "Gabriel Grace",
+    email: "gabrielgrace.com",
+    role: "HR Admin",
+    creationDate: "2009-09-04",
+    position: "Human Resource",
+    avatar: "images/img4.jpg",
+  },
+  {
+    id: 5,
+    name: "Tolulope Opeyemi",
+    email: "tolulopeopeyemi.com",
+    role: "Employee",
+    creationDate: "2022-03-14",
+    position: "Web designer",
+    avatar: "images/img3.png",
+  },
```

Comment:
Move this away.
- Create a new file within this component folder `users.metadata.js`
- Move the `employeesData` to this file and export
- Then import it into this component

Reply:
> 

@Doyin02 my instruction was "Create a new file **within this component folder** ✅ " NOT within the component folder ❌ 

Please pay careful attention to the things I write; and if you are not sure about something, feel free to reach out.